CHANGELOG.md

```diff
@@ -8,6 +8,16 @@ Mapbox welcomes participation and contributions from everyone.
 
 #### Bug fixes and improvements
 - :warning: Moved `MapboxAudioGuidanceButton` from Drop-in UI to Voice module. [#5874](https://github.com/mapbox/mapbox-navigation-android/pull/5874)
+- Added `Debug` logs to HistoryRecorder []()
+- Added explicitly fail EV onboard routing request []()
+
+- Added `Debug` logs to HistoryRecorder. [#5905](https://github.com/mapbox/mapbox-navigation-android/pull/5905)
```

Comment:
Something went wrong here. Also as mentioned before, let's remove the log entry.